_inc/client/components/jetpack-notices/index.jsx

```diff
@@ -182,36 +182,14 @@ UserUnlinked.propTypes = {
 	siteConnected: PropTypes.bool.isRequired,
 };
 
-export class ErrorNoticeCycleConnection extends React.Component {
-	static defaultProps = {
-		text: __( 'Connection Error, please reconnect.' ),
-	};
-
-	static propTypes = {
-		text: PropTypes.string.isRequired,
-	};
-
-	render() {
-		return (
-			<SimpleNotice
-				showDismiss={ false }
-				text={ this.props.text }
-				status={ 'is-error' }
-				icon={ 'link-break' }
-			>
-				<NoticeActionDisconnect>{ __( 'Reconnect' ) }</NoticeActionDisconnect>
-			</SimpleNotice>
-		);
-	}
-}
-
 class JetpackNotices extends React.Component {
 	static displayName = 'JetpackNotices';
 
 	render() {
 		return (
 			<div aria-live="polite">
 				<NoticesList />
+				{ this.props.siteConnectionStatus && <JetpackConnectionErrors /> }
```

Comment:
won't we always have this.props.siteConnectionStatus? Any reason not to get the connection errors here, and only load the component if errors are found?